### PR TITLE
Updated ppa link with Canonical's feedback

### PR DESCRIPTION
**Pull request by juhlee-microsoft**
The permanent location for the PPA is:
https://launchpad.net/~canonical-server/+archive/ubuntu/dpdk-azure-18.11
with updated libraries:
- rdma-core v24
- dpdk 18.11.2
- openvswitch 2.11.1

Azure-Functional-INFINIBAND.INFINIBAND-INTEL-MPI-2VM   - PASSED
Azure-Functional-INFINIBAND.INFINIBAND-MVAPICH-MPI-2VM   - PASSED
Azure-Functional-INFINIBAND.INFINIBAND-OPEN-MPI-2VM   - PASSED

### CODE REVIEW

**Comment by LiliDeng** [comment]
Did you run them against Ubuntu 16.04? I will run changes against other Ubuntu distro.

**Comment by juhlee-microsoft** [comment]
> Did you run them against Ubuntu 16.04? I will run changes against other Ubuntu distro.

Sure. If you do, I will check out the result of 18.04 and post here. Thank you.

**Comment by LiliDeng** [comment]
Canonical UbuntuServer 18.04-LTS latest
```
Azure-Functional-INFINIBAND.INFINIBAND-INTEL-MPI-2VM   - PASSED
Azure-Functional-INFINIBAND.INFINIBAND-MVAPICH-MPI-2VM   - PASSED
Azure-Functional-INFINIBAND.INFINIBAND-OPEN-MPI-2VM   - PASSED
```

**Review by LiliDeng — APPROVED** [review]

File: Testscripts/Linux/SetupRDMA.sh [hunk 1]
```diff
@@ -191,8 +191,8 @@ function Main() {
 			LogMsg "Starting RDMA setup for Ubuntu"
 			hpcx_ver="ubuntu"$VERSION_ID
 			LogMsg "Installing required packages ..."
-			LogMsg "*** Adding Canonical ppa for temporary fix"
-			add-apt-repository -y ppa:ci-train-ppa-service/3760
+			# the old fix integrated to dpdk-18.11 repo
+			add-apt-repository ppa:canonical-server/dpdk-azure-18.11 -y
```
Comment (LiliDeng) [inline Testscripts/Linux/SetupRDMA.sh:195]: We'd better to have a check here to make sure the ppa repo is added successfully, like this time, we need do much investigation if not catch this error earlier from log.

Comment (juhlee-microsoft) [reply Testscripts/Linux/SetupRDMA.sh:195]: fixed in ad6635b367739d7217fcb9c837f5d20730c3f85b

**Pull request merged**